<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>294</width>
          <height>421</height>
         </rect>
        </property>
        <property name="title">
         <string>Agregar</string>
        </property>
        <widget class="Line" name="line_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>20</width>
           <height>281</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>261</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>270</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Ord. Ensalada</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>150</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Pescuezos</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>190</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Ord. Papas</string>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>330</y>
           <width>261</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>30</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Sobrante</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>230</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Ord. Mole</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="pollo_pro_spinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>70</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="Line" name="line_3">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>60</y>
           <width>20</width>
           <height>281</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>310</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Chamorros</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tacos</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Producto</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Producido</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Pollos</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="taco_pro_spinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>110</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="pescuezo_pro_spinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>150</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="papa_pro_spinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>190</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="mole_pro_spinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>230</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="ensalada_pro_spinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>270</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="chamorro_pro_spinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>310</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="pollo_sob_spinBox">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>70</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="taco_sob_spinBox">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>110</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="pescuezo_sob_spinBox">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>150</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="papa_sob_spinBox">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>190</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="mole_sob_spinBox">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>230</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="ensalada_sob_spinBox">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>270</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="chamorro_sob_spinBox">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>310</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>350</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Fecha</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="agregar_fecha_checkBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>350</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QDateEdit" name="agregar_dateEdit">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>350</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximumDateTime">
          <datetime>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
           <year>2050</year>
           <month>12</month>
           <day>31</day>
          </datetime>
         </property>
         <property name="minimumDateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>1992</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="minimumDate">
          <date>
           <year>1992</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="date">
          <date>
           <year>2021</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
        </widget>
        <widget class="QPushButton" name="agregar_registro_pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>380</y>
           <width>271</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Agregar Registro</string>
         </property>
        </widget>
       </widget>
       <widget class="QTableWidget" name="agregar_tableWidget">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>20</y>
          <width>471</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="agregar_plainTextEdit">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>400</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>410</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Total:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Buscar y eliminar</string>
       </attribute>
       <widget class="QDateEdit" name="buscar_dateEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>110</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="buscar_tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>761</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>380</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buscar_pushButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>400</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="eliminar_pushButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>400</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>410</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Total:</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="buscar_plainTextEdit">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>400</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Cambiar precio</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>211</width>
          <height>411</height>
         </rect>
        </property>
        <property name="title">
         <string>Cambiar precio</string>
        </property>
        <widget class="Line" name="line_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>20</width>
           <height>301</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="Line" name="line_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>280</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Ord. Ensalada</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>160</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Pescuezos</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>200</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Ord. Papas</string>
         </property>
        </widget>
        <widget class="Line" name="line_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>350</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>240</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Ord. Mole</string>
         </property>
        </widget>
        <widget class="Line" name="line_8">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>60</y>
           <width>20</width>
           <height>301</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>320</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Chamorros</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>120</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tacos</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Producto</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Precio</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_23">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Pollos</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cambiar_precio_pushButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>370</y>
           <width>181</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Cambiar Precio</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="pollo_precio_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>80</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="taco_precio_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>120</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="pescuezo_precio_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>160</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="papa_precio_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>200</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="mole_precio_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>240</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="ensalada_precio_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>280</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="chamorro_precio_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>320</y>
           <width>62</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <double>300.000000000000000</double>
         </property>
        </widget>
       </widget>
       <widget class="QTableWidget" name="cambiar_precio_tableWidget">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>531</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="mostrar_precio_pushButton">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>390</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Mostrar precios</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
